From c7ad2523da88f057ae793e90603e2f1e2523cbbd Mon Sep 17 00:00:00 2001
From: Thomas d'Otreppe <tdotreppe@aircrack-ng.org>
Date: Sat, 2 May 2015 06:35:02 +0000
Subject: [PATCH] Airmon-ng: Fixed error when running 'check kill' due to a
 return when it should have been an exit (since it's not in a function).

git-svn-id: http://svn.aircrack-ng.org/trunk@2560 28c6078b-6c39-48e3-add9-af49d547ecab

Patch grabbed from upstream here:
https://github.com/aircrack-ng/aircrack-ng/commit/c7ad2523da88f057ae793e90603e2f1e2523cbbd

---
 scripts/airmon-ng | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/airmon-ng b/scripts/airmon-ng
index 02f25c2..ff0fe9a 100644
--- a/scripts/airmon-ng
+++ b/scripts/airmon-ng
@@ -975,7 +975,7 @@ then
 	then
 		#if we are killing, tell scanProcesses that
 		scanProcesses "${2}"
-		return
+		exit
 	elif [ x"${1}" = "xstart" ]
 	then
 		#this stub can send scanProcesses the interface name
